test/objectid.test.js

```diff
@@ -139,8 +139,8 @@ describe('ObjectID', function() {
       const found = await Article.find({where: {title: 'arrayOfObjectID'}});
       // the type of the returned array is actually string even it's stored as ObjectIds in the db as expected
       found[0].xidArr.should.containDeep([
-        new ds.ObjectID(objectIDLikeString),
-        new ds.ObjectID(objectIDLikeString2),
+        objectIDLikeString,
```

Comment:
This is the correct  expected object, refer to the code comment above.

Reply:
I feel this change is unrelated to dropping support for juggler v3, can we please remove it from this PR and land it as part of your ObjectID refactor?

Reply:
There is some sort of connection between the Juggler 3 tests and this particular test case, removing the Juggler 3 tests make this test case fail (as expected).

Reply:
I see. If the behavior is fixed now, then I think the comment no longer makes sense and should be removed?

Reply:
> I think the comment no longer makes sense and should be removed?

Which comment should be removed?

In case it was not clear, Juggler 3 tests were interfering in some way and causing the test case to pass when it should have actually failed because the array will contain string values not ObjectIds.

Reply:
I was referring to the following comment on line 140:

```js
// the type of the returned array is actually string even it's stored as ObjectIds in the db as expected
```

As I am re-reading it now, the comment seems valid, so I am fine to keep it.